packages/fetchai/protocols/fipa/dialogues.py

```diff
@@ -82,7 +82,7 @@ class FipaDialogue(Dialogue):
         ),
     }
 
-    class AgentRole(Dialogue.Role):
+    class Role(Dialogue.Role):
```

Comment:
Please regenerate all protocols with the current generator (but don't bump the versions). Otherwise the hashes won't be correct.